### DISCUSSION TITLE
Allow inspect access for read-only environments

### DISCUSSION THREAD

**Discussion by zslabs** [six7/figma-tokens#1284] [Ideas]
When designers are handing off things to an engineer, the plugin doesn't appear to show any information that would allow the engineer to inspect a design for the tokens used without requesting edit access in either the context-menu or inspect-pane:

<img width="1440" alt="Screen Shot 2022-09-14 at 2 42 44 PM" src="https://user-images.githubusercontent.com/1018402/190237065-e00265f4-665a-4c05-99f2-458cc67b8baf.png">

Could we surface a read-only view of the inspect tab in the interface? cc @six7

**Comment by zslabs** [comment]
Most likely opened this a bit early, as I saw that you can't run plugins in 'view-only' mode. There's probably not much we can do to get around that, but might be worth leaving open for any workaround suggestions.

**Comment by six7** [comment]
So this one is just not possible yet due to Figma constraints. Maybe it will be in the future, I'd certainly hope so. Guess we'll need to keep bugging Figma about this.
